src/js/views/hallOfFame.js

```diff
@@ -43,7 +60,32 @@ async function updatePlayers(inputs, updateEvents) {
     }
 }
 
-export default bbgmViewReact.init({
+function uiFirst(vm) {
```

Comment:
Just ignore this block for a moment. I fast-forwarded nearly 600 commits that happened since I worked on this branch and this is all old garbage that came from before the React rewrite.

This section is the only part that needs to be refactored.